App.tsx

```diff
@@ -1,14 +1,20 @@
 import React from 'react';
 import { NativeBaseProvider, StatusBar } from 'native-base';
+import { Provider } from 'react-redux';
 
-import { THEME } from './src/styles/theme';
-import Routes from './src/routes';
+import { THEME } from '@src/styles/theme';
+import Routes from '@src/routes';
+import store from '@src/store';
+
+const GRAY_600 = '#202024';
```

Comment:
I think it would be better if we use `src/styles/theme.ts` constants instead.

Reply:
I created a file to store all colors constants